test/util/util.go

```diff
@@ -0,0 +1,111 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	metallbv1beta1 "github.com/metallb/metallb-operator/api/v1beta1"
+	"github.com/metallb/metallb-operator/pkg/status"
+	"github.com/metallb/metallb-operator/test/consts"
+	testclient "github.com/metallb/metallb-operator/test/e2e/client"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// Timeout and Interval settings
+	Timeout  = time.Minute * 3
+	Interval = time.Second * 2
+)
+
+func decodeYAML(r io.Reader, obj interface{}) error {
+	decoder := yaml.NewYAMLToJSONDecoder(r)
+	return decoder.Decode(obj)
+}
+
+func loadMetalLBFromFile(metallb *metallbv1beta1.MetalLB, fileName string) error {
```

Comment:
Util functions should go on the bottom of the file, exposed ones on top

Reply:
done